### ISSUE TITLE
Cannot run install command 

### ISSUE THREAD

**Issue by bedmunds97** [davidteren/lpx_links#18]
It is coming up with: 'sudo: ./download_install.sh: command not found' Any thoughts?

**Comment by thekungfuspider** [comment]
Yes, the instructions were written by two people who may not have been in sync with one another. I created a fork of this repository and am in the process of fixing a couple things. @davidteren - I hope you're still around for review!

**Comment by davidteren** [comment]
@thekungfuspider please go ahead. :ok\_hand:

**Comment by xCessivePressure** [comment]
Hi,

Please forgive my noobishness, but I believe I have an issue related to this. 

As the command for download tells us, I've used -d to specify my prefered download folder. However, I cannot seem to use the install command because the files are not in the download folder. I've tried specifying the folder where i've downloaded everything, but I can't make it work. 

Help?

**Comment by xCessivePressure** [comment]
> Hi,
> 
> Please forgive my noobishness, but I believe I have an issue related to this.
> 
> I've used -d to specify my prefered download folder following the suggestion indicated in the download command. However, I cannot seem to make the install command work because the files are not in the default ~/Downloads/ folder. I've tried specifying the folder where i've downloaded everything, but I can't make it work.
> 
> Help?

Think i've just figured it out. Downloaded the download_install.sh and modified the command thus:

#!/bin/bash
cd /NAME OF FOLDER/
while read -r line; do
    name="$line"
    echo "Downloading $name..."
    curl -O $name
done < $1

for pkg in /NAME OF FOLDER/*.pkg; do
    echo "Installing $pkg file..."
    installer -pkg $pkg -target /
done


Then I copied and pasted in terminal.

**Comment by bedmunds97** [comment]
You guys are literal gods to me lmao, I'm so useless at computer stuff. So should that work now @AlftheElf if I copy your instructions into the terminal? Will that run the installation autonomously?

**Comment by AlftheElf** [comment]
> You guys are literal gods to me lmao, I'm so useless at computer stuff. So should that work now @AlftheElf if I copy your instructions into the terminal? Will that run the installation autonomously?

Yes it should install autonomously. Have you successfully downloaded the packages using aria2c first?

Also, don't copy this script into the terminal:

```
#!/bin/bash
for pkg in $1/*.pkg; do
    echo "Installing $pkg file..."
    installer -pkg $pkg -target /
done
```

You need to edit the `~/Downloads/lpx_links/app/download_install.sh` file for that part. Then yes, run `sudo ~/Downloads/lpx_links/app/download_install.sh ~/Downloads/logic_content` and it should install everything.

**Comment by mkman7** [comment]
Hi,
I have installation problem,
But I also tried to install pkg file by run it manually, then I got another error msg.
any ideas? Thx.

![Screen Shot 2023-04-15 at 13 40 58](https://user-images.githubusercontent.com/20065574/232209157-85c7023c-37d4-470a-bd57-1af51fbbd42e.png)

![Screen Shot 2023-04-15 at 13 38 44](https://user-images.githubusercontent.com/20065574/232209061-bdc9097e-6509-407b-a691-5e23b977e0a5.png)

**Comment by AlftheElf** [comment]
@mkman7 Did you edit the download_install.sh before running it? When I ran the original version it gave me the same error you have, and can’t even be installed manually.

If you replace all the code in the script with this,

```
#!/bin/bash
for pkg in $1/*.pkg; do
    echo "Installing $pkg file..."
    installer -pkg $pkg -target /
done
```

it won’t run the part which corrupts the packages.
If you already ran the download_install.sh script without editing it first, you will have to download everything again, which is what I had to do.

**Comment by mkman7** [comment]
I got it now. 
Thank you !

**Comment by davidteren** [comment]
Thanks to @AlftheElf this [PR](https://github.com/davidteren/lpx_links/pull/24) should solve the issue.
